drivers/hmis_external_apis/app/models/hmis_external_apis/mci_payload.rb

```diff
@@ -0,0 +1,144 @@
+            'schoolDistrictCode' => '12',
+            'stateCode' => 'AZ',
+            'zipCode' => '11111',
+            'zipCodeExt' => 'NDZC',
+            'countryCode' => 97,
+            'startDate' => '2019-05-09T11:23:31.128Z',
+          },
+          {
+            'addressLine1' => '423 Broadway Mailing',
+            'addressLine2' => 'unit 3',
+            'addressTypeCode' => 2,
+            'aptNumber' => '102',
+            'city' => 'Cool',
+            'cityCouncilDistrictCode' => 2,
+            'countyCode' => 2,
+            'countyCouncilDistrictCode' => 2,
+            'isCurrentResidence' => true,
+            'isVerified' => true,
+            'municipalityCode' => 2,
+            'schoolDistrictCode' => '2',
+            'stateCode' => 'AZ',
+            'zipCode' => '66212',
+            'zipCodeExt' => '1234',
+            'countryCode' => 37,
+            'startDate' => '2019-05-09T11:23:31.128Z',
+          },
+          {
+            'addressLine1' => '515 High Street OTHERU',
+            'addressLine2' => 'Near Opera House OTHERU',
+            'addressTypeCode' => 3,
+            'aptNumber' => '1093',
+            'city' => 'Cool',
+            'cityCouncilDistrictCode' => 3,
+            'countyCode' => 3,
+            'countyCouncilDistrictCode' => 3,
+            'isCurrentResidence' => true,
+            'isVerified' => true,
+            'municipalityCode' => 3,
+            'schoolDistrictCode' => '3',
+            'stateCode' => 'NY',
+            'zipCode' => 'NYZ',
+            'zipCodeExt' => 'NYZE',
+            'countryCode' => 90,
+            'startDate' => '2019-05-09T11:23:31.128Z',
+          },
+        ],
+        'contact' => {
+          'homePhone' => '5655652377',
+          'faxNumber' => '5655652504',
+          'email' => 'contacttest1@gmail.com',
+          'workPhone' => '565565',
+          'workPhoneExt' => '33333',
+          'cellPhone' => '7045652777',
+        },
+        'residencyCode' => 3,
+        'firstName' => client.first_name,
+        'middleName' => client.middle_name,
+        'lastName' => client.last_name,
+        'suffix' => client.name_suffix,
+        'ssn' => client.ssn,
+        'birthDate' => client.dob.to_s(:db),
+        'raceCodes' => '2-,',
+        'deathDate' => '2019-05-09T11:23:31.129Z',
+      }
+
+      # if true # FIXME: birth name exists
+      #   payload['birthName'] = {
+      #     'firstName' => client.first_name,
+      #     'middleName' => client.middle_name,
+      #     'lastName' => client.last_name,
+      #     'suffix' => client.name_suffix,
+      #     'dateVerified' => '2019-05-09T11:23:31.127Z',
+      #     'dateUpdated' => '2019-05-09T11:23:31.127Z',
+      #   }
+      # end
+
+      # if false # FIXME: get external id if we have it
+      #   # FIXME: pseudo-code:
+      #   # payload["mciId"] => client.external_ids.mci_id
+      # end
+
+      payload
+    end
+
+    # FIXME: what do we need?
```

Comment:
First, last, middle, suffix, dob, and SSN would be a good starting point. I can add in the reverse race/ethnicity/gender mappings after.